app/logic/events.py

```diff
@@ -227,7 +227,7 @@ def getOtherEvents(term):
                             .where(Event.term == term,
                                    Event.isTraining == False,
                                    Event.isAllVolunteerTraining == False,
-                                   ((Event.program == None) |
+                                   ((Event.program_id == 9) |
```

Comment:
Like with Bonner events above, you can change `Event.program_id == 9` to `Event.isCeltsSponsored`